dlgr/griduniverse/bots.py

```diff
@@ -306,6 +307,13 @@ def complete_questionnaire(self):
         difficulty.select_by_value(str(random.randint(1, 7)))
         engagement = Select(self.driver.find_element_by_id('engagement'))
         engagement.select_by_value(str(random.randint(1, 7)))
+        try:
+            fun = Select(self.driver.find_element_by_id('fun'))
+            # This is executed by the IEC_demo.py script...
+            # No need to fill out a random value.
+            fun.select_by_value(str(0))
```

Comment:
This can be improved by getting the fun feedback from `iec_demo.py`... Now i'm logging the database with 0 because I'm not sure how to get the script to send the fun feedback back to the `bot` questionnaire behavior, and my script doesn't use that database value for the bots. This might be a trivial problem since the bot feedback doesn't actually need to be in the database, as long as the player data gets inputted correctly (which it should).